<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-65LALTP" failures="1" tests="2" name="GRP-reordersignin price,reordersignin" time="242.287" errors="0" timestamp="2023-03-19T18:04:26 EET">
  <testcase classname="project_final.home_page" name="t_5CHECKOUTPAGE" time="148.141"/>
  <testcase classname="project_final.home_page" name="t_6PriceREorder" time="94.108">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.testng.asserts.SoftAssert.assertEquals(int, int)&amp;quot; because &amp;quot;this.myAssertion&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.testng.asserts.SoftAssert.assertEquals(int, int)" because "this.myAssertion" is null
at project_final.home_page.t_6PriceREorder(home_page.java:363)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- t_6PriceREorder -->
  <testcase classname="project_final.home_page" name="t_4CHECKOUTPAGEnew" time="0.0">
    <ignored/>
  </testcase> <!-- t_4CHECKOUTPAGEnew -->
  <testcase classname="project_final.home_page" name="t_3ADDITEM" time="0.0">
    <ignored/>
  </testcase> <!-- t_3ADDITEM -->
  <testcase classname="project_final.home_page" name="t_2searchbar" time="0.0">
    <ignored/>
  </testcase> <!-- t_2searchbar -->
  <testcase classname="project_final.home_page" name="t_1Signuptestvalidpassword" time="0.0">
    <ignored/>
  </testcase> <!-- t_1Signuptestvalidpassword -->
</testsuite> <!-- GRP-reordersignin price,reordersignin -->
